src/app/screens/Home/index.js

```diff
@@ -7,8 +7,9 @@ import { arrayOf, bool, string } from 'prop-types';
 import { objectIsEmpty } from '@widergy/web-utils/lib/object';
 import i18 from 'i18next';
 import DescriptionIcon from '@material-ui/icons/Description';
+import PaymentIcon from '@material-ui/icons/Payment';
 
-import { BILLS_HISTORY } from 'constants/routes';
+import { BILLS_HISTORY, LIST_OF_PAYMENTS } from 'constants/routes';
```

Comment:
```suggestion
import { BILLS_HISTORY, PAYMENTS_HISTORY } from 'constants/routes';
```
Para continuar con la nomenclatura utilizada.